docs/kubernetes/getstarted.md

```diff
@@ -0,0 +1,604 @@
+
+## Provisioning
+
+To provision our VMs we first need to setup a network between different Nodes on the TFGrid
+We will use devnet Nodes to select some nodes go to :[devnet Cockpit](https://cockpit.devnet.grid.tf/)
+We picked three nodes identified by these NodeId
+
+- nodeid1: FTothsg9ZuJubAEzZByEgQQUmkWM637x93YH1QSJM242
+- nodeid2: 3NAkUYqm5iPRmNAnmLfjwdreqDssvsebj4uPUt9BxFPm
+- nodeid3: 2anfZwrzskXiUHPLTqH1veJAia8G6rW2eFLy5YFMa1MP
+
+Becarefull NodeId is case sensitive
+
+### Networking
+
+We now need to create a network between the nodes. Hopefully tfuser can help us generate the configuration file.
+Let's first create a network named `kubetest` with a large cidr `173.30.0.0/16` and write that configuration inside a file `network.json`
+
+```
+$ ./tfuser generate --schema network.json network create --name kubetest --cidr 173.30.0.0/16
+```
+
+now let's add the nodes that we have selected inside our network by specifying the path to our `network.json` network configuration file. We also specify a port for wireguard to use and a subnet to use on the nodes.
+
+```
+$ ./tfuser generate --schema network.json network add-node --node qzuTJJVd5boi6Uyoco1WWnSgzTb7q8uN79AjBT9x9N3  \
+				      --subnet 173.30.1.0/24
+$ ./tfuser generate --schema network.json network add-node --node 3NAkUYqm5iPRmNAnmLfjwdreqDssvsebj4uPUt9BxFPm \
+				      --subnet 173.30.2.0/24
+$ ./tfuser generate --schema network.json network add-node --node FTothsg9ZuJubAEzZByEgQQUmkWM637x93YH1QSJM242 \
+				      --subnet 173.30.3.0/24
+```
+
+there are two levels here :
+
+- one for the global network with a cidr /16
+- one for network resources on the node with a cidr /24. All workloads in this network resource will receive an IP from this subnet. The subnet must be part of the CIDR provided when the network is created.
+
+**NOTE** that you can't give the same subnet to all of your nodes. Each node must be assigned a different /24 subnet that is part of the global /16 of the network.
+
+We will need an external access to the network to be able to ssh into our VMs.
+
+```
+$ ./tfuser generate --schema network.json network add-access --node qzuTJJVd5boi6Uyoco1WWnSgzTb7q8uN79AjBT9x9N3 \
+				      --subnet 173.30.4.0/24 --ip4 > wg.conf
+```
+
+We generated the wireguard, a secure and fast VPN, configuration file that we will use here after in this guide.
+
+Let's take a look at a graphical representation of our network
+
+```
+$ ./tfuser generate --schema network.json network graph
+```
+
+This will generate network.json.dot file. You can copy paste the content on a website like [graphiz](https://dreampuf.github.io/GraphvizOnline) to see the visual representation of the network we defined.
+
+![network graphical representation](network_dot_file.png)
+
+Now we have a nodes in a network definition ready to be provisioned. Let's use our seed to sign the transaction and ask for a 2 hours provision.
```

Comment:
`--duration 2` doesn't reserve for 2 hours but for 2 days. If you want 2hours then you need to do `--duration 120m`

Reply:
mistake from copy paste .. corrected